src/components/Articles.jsx

```diff
@@ -1,21 +1,43 @@
 import React, { useEffect,useState,useContext } from "react"
+import {useParams} from "react-router-dom"
 import axios from 'axios'
+import { getArticles } from "../utils/api"
 
 const Articles = () => {
+   
     const[articles, setArticles] = useState([])
-}
+
 
 useEffect(() => {
-    axios.get("")
-    .then(({res})=> {
-       return res.json();
-    })
-    .then((data) => {
-        setArticles(data.articles)
-    })
-})
+   getArticles().then((articles) => {
+   setArticles(articles)
+   })
+},[])
+
 
 
     return (
-        <div className="body">
-                    </div> )
\ No newline at end of file
+       <div>
+        <h1>Articles</h1>
+       {articles.map((article) => {
+        return (
+            <div key={article.article_id}> 
+            <h2>
+             {article.title}
+            <p>{article.topic}</p>
+            <p>{article.author}</p>
+            <p> {article.body}</p>
+            <p>{article.created_at}</p>
+            <p> {article.votes}</p>
+             <img src={article.article_img_url} alt="Article Image" />
+             
```

Comment:
This design isn't currently responsive (so when you use the site on a mobile phone you have to scroll left and right to see all the content). Can you please implement some css that makes this site responsive please 📱